File: articles/algorithmic-accountability-and-audit-trails/haskell/AccountabilityCapacityScore.hs
module Main where

main :: IO ()
main = do
  let scores = [0.72, 0.68, 0.64, 0.58, 0.52, 0.66]
      capacity = sum scores / fromIntegral (length scores)
  putStrLn ("accountability_capacity_score=" ++ show capacity)

File: articles/algorithmic-accountability-and-audit-trails/typescript/accountability_capacity_score.ts
const scores = [0.72, 0.68, 0.64, 0.58, 0.52, 0.66];
const capacity = scores.reduce((a, b) => a + b, 0) / scores.length;
console.log(`accountability_capacity_score=${capacity.toFixed(4)}`);
